src/components/VideoPlayerPreview/VideoPlayerThumbnail.tsx

```diff
@@ -13,19 +13,13 @@ import * as ReportUtils from '@libs/ReportUtils';
 import variables from '@styles/variables';
 import CONST from '@src/CONST';
 
-const propTypes = {
-    onPress: PropTypes.func.isRequired,
-
-    accessibilityLabel: PropTypes.string.isRequired,
-
-    thumbnailUrl: PropTypes.string,
-};
-
-const defaultProps = {
-    thumbnailUrl: undefined,
+type VideoPlayerThumbnailProps = {
+    thumbnailUrl: string | undefined;
```

Comment:
```suggestion
    thumbnailUrl?: string;
```

Wouldn't be the same?